docker-bake.hcl

```
@@ -0,0 +1,113 @@
+group "linux" {
+  targets = [
+    "alpine_jdk8",
+    "alpine_jdk11",
+    "debian_jdk8",
+    "debian_jdk11",
+  ]
+}
+
+group "linux-arm64" {
+  targets = [
+    "debian_jdk11",
+  ]
+}
+
+group "linux-s390x" {
+  targets = [
+    "debian_jdk11",
+  ]
+}
+
+group "linux-ppc64le" {
+  targets = [
+    "debian_jdk11",
+  ]
+}
+
+group "windows" {
+  targets = [
+    "windows_2019_jdk11",
+  ]
+}
+
+variable "REMOTING_VERSION" {
+  default = "4.10"
+}
+
+variable "REGISTRY" {
+  default = "docker.io"
+}
+
+variable "JENKINS_REPO" {
+  default = "jenkins/inbound-agent"
+}
+
+variable "BUILD_NUMBER" {
+  default = "1"
+}
+
+variable "ON_TAG" {
+  default = "false"
+}
+
+target "alpine_jdk8" {
+  dockerfile = "8/alpine/Dockerfile"
+  context = "."
+  args = {
+    REMOTING_VERSION = REMOTING_VERSION
+  }
+  tags = [
+    equal(ON_TAG, "true") ? "${REGISTRY}/${JENKINS_REPO}:${REMOTING_VERSION}-${BUILD_NUMBER}-alpine-jdk8": "",
+    "${REGISTRY}/${JENKINS_REPO}:alpine-jdk8",
+    "${REGISTRY}/${JENKINS_REPO}:latest-alpine-jdk8",
+  ]
+  platforms = ["linux/amd64"]
+}
+
+target "alpine_jdk11" {
+  dockerfile = "11/alpine/Dockerfile"
+  context = "."
+  args = {
+    REMOTING_VERSION = REMOTING_VERSION
+  }
+  tags = [
+    equal(ON_TAG, "true") ? "${REGISTRY}/${JENKINS_REPO}:${REMOTING_VERSION}-${BUILD_NUMBER}-alpine": "",
+    equal(ON_TAG, "true") ? "${REGISTRY}/${JENKINS_REPO}:${REMOTING_VERSION}-${BUILD_NUMBER}-alpine-jdk11": "",
+    "${REGISTRY}/${JENKINS_REPO}:alpine",
+    "${REGISTRY}/${JENKINS_REPO}:alpine-jdk11",
+    "${REGISTRY}/${JENKINS_REPO}:latest-alpine",
+    "${REGISTRY}/${JENKINS_REPO}:latest-alpine-jdk11",
+  ]
+  platforms = ["linux/amd64"]
+}
+
+target "debian_jdk8" {
+  dockerfile = "8/debian/Dockerfile"
+  context = "."
+  args = {
+    REMOTING_VERSION = REMOTING_VERSION
+  }
+  tags = [
+    equal(ON_TAG, "true") ? "${REGISTRY}/${JENKINS_REPO}:${REMOTING_VERSION}-${BUILD_NUMBER}-jdk8": "",
+    "${REGISTRY}/${JENKINS_REPO}:jdk8",
+    "${REGISTRY}/${JENKINS_REPO}:latest-jdk8",
+  ]
+  platforms = ["linux/amd64"]
+}
+
+target "debian_jdk11" {
+  dockerfile = "11/debian/Dockerfile"
+  context = "."
+  args = {
+    REMOTING_VERSION = REMOTING_VERSION
+  }
+  tags = [
+    equal(ON_TAG, "true") ? "${REGISTRY}/${JENKINS_REPO}:${REMOTING_VERSION}-${BUILD_NUMBER}": "",
+    equal(ON_TAG, "true") ? "${REGISTRY}/${JENKINS_REPO}:${REMOTING_VERSION}-${BUILD_NUMBER}-jdk11": "",
+    "${REGISTRY}/${JENKINS_REPO}:jdk11",
+    "${REGISTRY}/${JENKINS_REPO}:latest",
+    "${REGISTRY}/${JENKINS_REPO}:latest-jdk11",
+  ]
+  platforms = ["linux/amd64", "linux/arm64"]
```

you added this image to the groups for the other platforms but didn't add them here was there a reason? (I assume needs https://github.com/jenkinsci/docker-agent/pull/185)
```suggestion
  platforms = ["linux/amd64", "linux/arm64", "linux/ppc64le", "linux/s390x"]
```

---

Yes, the ppc64le and s390x images are not yet published, so I didn't include them here.

---

they will be published before this is built so I would include them now